### PR TITLE
Fix/get bind missing kwarg

### LINKED ISSUE #942: get_bind function missing 'bind' kwarg in SignallingSession class
Hi there, a colleague and I have been working on using Flask-Sqlalchemy for a new project. We've been needing to run raw queries against two separate databases.
Based on the info we could find, we were executing these in the following way:

```
results_a = db.session.execute(query, bind_arguments={'bind': db.get_engine(app, 'database_a')})
results_b = db.session.execute(query, bind_arguments={'bind': db.get_engine(app, 'database_b')})
```

We ran into an interesting issue where we were getting the error: `get_bind() got an unexpected keyword argument 'bind'`

We found that the execute function was defined in: sqlalchemy/orm/session.py:1522

It specifies the following params for bind_agruments
`
:param bind_arguments: dictionary of additional arguments to determine the bind.  May include "mapper", "bind", or other custom arguments. Contents of this dictionary are passed to the :meth:`.Session.get_bind` method.
`
And eventually calls the session get_bind function on line 1659.

We hit this issue because the `SignallingSession` `get_bind` doesn't accept the `bind` kwarg.

Environment:

- Python version: 3.8
- Flask-SQLAlchemy version: 2.5.1
- SQLAlchemy version: 1.4.3

### PR DESCRIPTION
**Pull request by alistairwoodcock**
Hello, I reported the issue #942. I've got a simple fix for it. I followed the instructions in contributing and branched off of 2.x but that caused some issues where I wasn't able to properly run some items on the checklist. 

I understand that there's an unrelease 3.0.0 version, so there may be some issues with going off of 2.x. I've spent more time than I have trying to resolve this issue and I hope where I've got this PR is an acceptable state, if not I can fix anything requested up, but I may need more specific instructions. Python and Flask are not my area of expertise. 

- fixes #942 

Checklist:

- [x] Add tests that demonstrate the correct behavior of the change. Tests should fail without the change.
- [x] Add or update relevant docs, in the docs folder and in code.
- [ ] Add an entry in `CHANGES.rst` summarizing the change and linking to the issue.
- [ ] Add `.. versionchanged::` entries in any relevant code docs.
- [x] Run `pre-commit` hooks and fix any issues.
- [x] Run `pytest` and `tox`, no tests failed.

### CODE REVIEW

File: docs/binds.rst [hunk 1]
```diff
@@ -71,3 +71,12 @@ a table object directly you will have to put it in there::
 If you specified the `__bind_key__` on your models you can use them exactly the
 way you are used to.  The model connects to the specified database connection 
 itself.
+
+Calling Binds with a Raw Query
```
Comment (davidism) [inline docs/binds.rst:75]: This is too technical to need inclusion in the docs.

File: flask_sqlalchemy/__init__.py [hunk 2]
```diff
@@ -184,7 +184,7 @@ def __init__(self, db, autocommit=False, autoflush=True, **options):
             bind=bind, binds=binds, **options
         )
 
-    def get_bind(self, mapper=None, clause=None):
+    def get_bind(self, mapper=None, clause=None, bind=None):
```
Comment (davidism) [inline flask_sqlalchemy/__init__.py:187]: This should be `**kwargs`, as there are other kwargs that are not included.

File: flask_sqlalchemy/__init__.py [hunk 3]
```diff
@@ -184,7 +184,7 @@ def __init__(self, db, autocommit=False, autoflush=True, **options):
             bind=bind, binds=binds, **options
         )
 
-    def get_bind(self, mapper=None, clause=None, bind=None):
+    def get_bind(self, mapper=None, clause=None, **kwargs):
```
Comment (davidism) [inline flask_sqlalchemy/__init__.py:187]: Since our function doesn't use `clause`, we can remove that to be handled by `kwargs`.

**Comment by james-willcox** [comment]
This is good, need this

**Comment by alistairwoodcock** [comment]
Is there anything more you need me to do to help resolve those unsuccessful checks?

**Comment by davidism** [comment]
It seems like no checks are running. Make sure you're committing against the latest master.

**Pull request merged**